collect_app/src/main/java/org/odk/collect/android/utilities/ServerPollingJobCreator.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2018 Nafundi
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.odk.collect.android.utilities;
+
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import com.evernote.android.job.Job;
+import com.evernote.android.job.JobCreator;
+
+import org.odk.collect.android.tasks.ServerPollingJob;
+
+public class ServerPollingJobCreator implements JobCreator {
+
+    @Override
+    @Nullable
+    public Job create(@NonNull String tag) {
+        switch (tag) {
```

Comment:
Using a switch statement seems a bit heavy weight, could use an if-else or a ternary here.

Reply:
I used the example from https://github.com/evernote/android-job but I agree that in this case, we can improve the code. FIxed.